extensions/amp-story-panning-media/0.1/amp-story-panning-media.md

```diff
@@ -0,0 +1,191 @@
+---
+$category@: media
+formats:
+  - stories
+teaser:
+  text: Fill this in with teaser text to improve SEO. Use the component description.
+draft: true
+tags:
+  - image
+  - transition
+  - animaion
+  - media
+author: philipbell
+toc: true
+$title: amp-story-panning-media
+version: '0.1'
+versions:
+  - '0.1'
+latest_version: '0.1'
+is_current: true
+$path: /documentation/components/amp-story-panning-media.html
+$localization:
+  path: '/{locale}/documentation/components/amp-story-panning-media.html'
+scripts:
+  - >-
+    <script async custom-element="amp-story-panning-media"
+    src="https://cdn.ampproject.org/v0/amp-story-panning-media-0.1.js"></script>
+layouts:
+  - Fill
```

Comment:
We would need to change the supported layouts here or in the actual extension code (in the JS we are only allowing `fill`).

EDIT: just realized this is scaffolding markdown. Would it be better to just include an empty file in that case? Would make it easier to review when actual information is written.